docker_build/build.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
why not move all this inside the Dockerfile itself. when we compose we will have all the dependency ready. So, only the build scripts can be part of the execution. its too slow to do this each time.

Reply:
Moved all dependencies to Dockerfile. Now when we re-run the container, it simply executes the build scripts.